--- /dev/fd/11	2026-07-19 01:30:13
+++ /dev/fd/12	2026-07-19 01:30:13
@@ -22,11 +22,13 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "import": "./dist/index.js"
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
     },
     "./v1": {
       "types": "./dist/v1/index.d.ts",
-      "import": "./dist/v1/index.js"
+      "import": "./dist/v1/index.js",
+      "default": "./dist/v1/index.js"
     }
   },
   "dependencies": {
